pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1194,6 +1193,12 @@
       ^-  move
       [duct %pass /channel/timeout/[channel-id] %b %wait expiration-time]
     ::
+    ++  heartbeat-move
+      |=  [channel-id=@t]
+      ^-  move
+      :^  duct  %pass  /channel/heartbeat/[channel-id]
+      [%b %wait (add now ~s20)]
```

Comment:
You'll probably want a matching `+cancel-heartbeat-move` for when a channel closes so there isn't a queued heartbeat. Yes, you check for the existence of the channel in `+on-channel-heartbeat`, but better to never receive the wakeup in the first place.